src/Domain/Creators/BinaryCompactObject.cpp

```diff
@@ -300,7 +298,7 @@ BinaryCompactObject::initial_refinement_levels() const noexcept {
   // domain--unless one or both of the interiors are not excised. (For each
   // interior not excised, there is one extra block appended to the list of
   // CoordinateMaps.)
-  if (addition_to_outer_layer_radial_refinement_level_) {
+  if (static_cast<bool>(addition_to_outer_layer_radial_refinement_level_)) {
```

Comment:
Why not just check if it's greater than zero? That would be easier to read

Reply:
I did that originally but another reviewer asked for this way. I have no preference

Reply:
I didn't realize there would be a `static_cast<bool>` but now I understand why it would be there - I agree that simply having `> 0` is better.